web/themes/custom/recipe_theme/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Ignore SASS files
+/.sass-cache
+.DS_Store
+*.codekit
+.idea
+node_modules
+/assets/
```

Comment:
This is a good first step, however the assets are still under version control. To remove them from VC, but keep it on your local (which doesn't really matter as running `yarn run watch` regenarates this folder anyway) you should also run:

```shell
git rm --cached assets
```